### ISSUE TITLE
ch-run (--version|--help) cannot be called as privileged user

### ISSUE THREAD

**Issue by olifre** [hpc/charliecloud#71]
This seems to be a design feature, but a bit non-intuitive in the case charliecloud is installed on the host and a privileged user wants to check e.g. the installed version. 

In real life, I hit this when trying to implement charliecloud support in HTCondor. 
HTCondor's daemons usually run privileged, and it needs some way to check whether charliecloud is set up correctly, and advertise the version (so job submitters can e.g. request to only run on nodes with version 0.xy). 
This fails, since calling `ch-run --version` fails when run as root. 

So in short: 
It would be best if arguments which will not lead to actual container execution (`--help`, `--version`) would also work for privileged users.

**Comment by reidpr** [comment]
This seems like a valid usability issue.

While non-setuid mode lets UID 0 run the program, in practice it may still error out because GID 0 is not permitted.

I see there's a PR now, so I'll make technical comments there.
